cnf-certification-test/suite_test.go

```diff
@@ -116,9 +116,32 @@ func loadJUnitXMLIntoMap(result map[string]interface{}, junitFilename, key strin
 	}
 }
 
-// SetLogFormat sets the log format for logrus
-func SetLogFormat() {
-	log.Info("debug format initialization: start")
+// logLevel retrieves the TNF_LOG_LEVEL environment variable
+func logLevel() string {
+	logLevel := os.Getenv("TNF_LOG_LEVEL")
```

Comment:
I think this env var's value was actually read and saved by the configuration.LoadEnvironmentVariables(). I've [refactored ](https://github.com/test-network-function/cnf-certification-test/pull/115/files#diff-6a029f2cd4126be02ace9b8e7e8c25c999fadc6aa23fb78304b414078f0a1ae9)it a little bit in #115, but it's still pending for review.

Reply:
Done.